ui/components/component-library/modal-body/README.mdx

```diff
@@ -0,0 +1,80 @@
+import { Controls, Canvas } from '@storybook/blocks';
+
+import * as ModalBodyStories from './modal-body.stories';
+
+# ModalBody
+
+`ModalBody` is a container component that handles the scrolling of the `Modal` content.
+
+<Canvas of={ModalBodyStories.DefaultStory} />
+
+## Props
+
+The `ModalBody` accepts all props below as well as all [Box](/docs/components-ui-box--default-story#props) component props
+
+<Controls of={ModalBodyStories.DefaultStory} />
```

Comment:
Uses `Controls` instead of `ArgsTable` which is deprecated 
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#argstable-block